examples/amm/amm.py

```diff
@@ -37,6 +37,23 @@
 # WARNING: This code is provided for example only. Do NOT deploy to mainnet.
 
```

Comment:
@barnjamin If you're not opposed, can a `pragma` declaration be added?

I opened https://github.com/algorand-devrel/beaker/pull/98 to illustrate the change.